### PR TITLE
Update README based on ASC 2019 Q4 Meeting

### PR DESCRIPTION
**Pull request by SteVwonder**
Add ASC membership and leadership.

Include for convenience the voting process visualizations made by @jjhursey.

### CODE REVIEW

**Comment by jjhursey** [comment]
For the process images, you can remove the red circle with a number in it. That was to indicate how many PRs we have in that state at the moment. So not something we need in the generall version.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,72 @@ Rules used by the Administrative Steering Committee (ASC) for the PMIx Standard.
...
+* Release Managers for PMIx v5
+  * Ken Raffenetti, ANL
+  * David Solt, IBM
+
+## Membership
+
+* Altair
+  * Michael Karo (Rep.)
+* Argonne
+  * Ken Raffenetti (Rep.)
```
Comment (jjhursey) [inline README.md]: Add `Shane Snyder (Alt.)` for Argonne

**Comment by naughtont3** [comment]
@SteVwonder I added updates from @jjhursey as PR on this PR.  (I wasn't sure of proper procedure.)

**Comment by SteVwonder** [comment]
Thanks @naughtont3! I just gave you collaboration access to my fork if you want to directly push in the future.

**Comment by naughtont3** [comment]
OK, thanks @SteVwonder

**Pull request merged**